Makefile

```diff
@@ -546,6 +546,24 @@ else
 	echo "Versions are different across Makefiles. Please ensure to keep them uniform."
 endif
 
+
+.PHONY: install-yamllint
```

Comment:
The installation way needs to change. refer - https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/hack/tools/Makefile#L59-L61

Reply:
@Karthik-K-N @Amulyam24 

also I am wondering if we should explore on reliability of yamllint source.

For ref kubebuilder - https://github.com/kubernetes-sigs/kubebuilder/blob/master/.github/workflows/lint.yml#L29-L34

For ref test-infra - https://github.com/kubernetes/test-infra/blob/master/hack/make-rules/verify/yamllint.sh

Reply:
Good point @Prajyot-Parab, [wasilibs/go-yamllint](https://github.com/wasilibs/go-yamllint/) looks like it is recently developed, not sure how well it will be maintained going further. Do we want to explore with a another reliable source?

Reply:
@Amulyam24 IMO yes we should try out something more stable/matured source.

Reply:
@priyanshikhetwani, initially [adrienverge/yamllint](https://github.com/adrienverge/yamllint) was being considered, were there any challenges with it which made you switch to [go-yamllint](https://github.com/wasilibs/go-yamllint/) ?

Reply:
@Amulyam24 @Prajyot-Parab I primarily switched because adrienverge/yamllint requires Python and pip.
I am exploring other better/mature options as well.